automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingApiSpec.scala

```diff
@@ -105,7 +105,8 @@ class BillingApiSpec extends FreeSpec with BillingFixtures with MethodFixtures w
           SimpleMethodConfig.rootEntityType,
           participantId,
           "this",
-          useCallCache = false)
+          useCallCache = false,
+          deleteIntermediateOutputFiles = false)
```

Comment:
[TOL / bikeshedding] When we talk about this feature in English, we refer to it as `deleteIntermediates`... maybe that concise name should make it into the code as well?

Reply:
That's the the name chosen in cromwell. Just plumbing it through. We can update it there though and submit a future PR here?